QuizApp/Login/LoginDatasource.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+class LoginDatasource {
```

Comment:
For each implementation, you need to have a protocol.
This goes for usecases, repositories, datasources and clients.

Reply:
https://github.com/barbarakos/QuizApp/pull/8